examples/wdio/mocha/wdio.conf.js

```diff
@@ -4,8 +4,9 @@ exports.config = {
     /**
      * server configurations
      */
-    hostname: 'localhost',
-    port: 4444,
+    // hostname: 'localhost',
+    // port: 4444,
+    // path: '/wd/hub',
```

Comment:
was this commented out for testing purposes or because this is no longer needed with v6?

Reply:
yeah, for testing purposes, will remove